variantgrid/settings/components/default_settings.py

```diff
@@ -383,7 +386,7 @@
 
 GENERATED_DIR = os.path.join(MEDIA_ROOT, 'generated')
 
-HGVS_DEFAULT_METHOD = "pyhgvs"  # HGVSConverterType (any case) ie "pyhgvs", "biocommons_hgvs"
+HGVS_DEFAULT_METHOD = "combo"  # HGVSConverterType (any case) ie "pyhgvs", "biocommons_hgvs", "combo"
```

Comment:
Just confirming we still want combo as the default method and not biocommons_hgvs? (ignore if this is already as desired).

Reply:
Yeah this was accidental will only switch once all hgvs issues fixed